.github/FUNDING.yml

```
@@ -1,2 +1,2 @@
-github: techwithanirudh
-buy_me_a_coffee: techwithanirudh
+github: techwithanirudh
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)